apps/nestjs-indexer/src/services/buy-token/buy-token.service.ts

```diff
@@ -138,6 +143,8 @@ export class BuyTokenService {
           },
         });
       });
+
+      await this.candlestickService.generateCandles(data.memecoinAddress, 5);
```

Comment:
We need to do await this because it's gonna blocked the indexer and overwhelmed the async between each event and block loading issue